### ISSUE TITLE
[ENHANCEMENT] Wyze Cam Support for Prusa Link/Connect

### ISSUE THREAD

**Issue by chrishilbert** [prusa3d/Prusa-Link-Web#436]
Please integrate with the Wyze APIs for camera support.  A large portion of the community is using Wyze cams because they are cheap and high quality to monitor our Pursa printers.

https://support.wyze.com/hc/en-us/articles/16129834216731-Creating-an-API-Key

**Comment by bjohns029** [comment]
This would be very useful!

**Comment by Mr3Deee** [comment]
Agreed! It is somehow really sad to see that fiddling around with an additional phone or a Pi is needed to get this done on a 4000$ printer. Setting up something which is already in use (if not already built in) sounds like what I'd expect.

**Comment by ErikV1000** [comment]
I seccond that. I have been using Wyze cams for the last few years to monitor my printers and in the last upgrade round (MK3S+/MMU3's and MK4) started using Prusa connect. It would be great to be able to connect my camera's into the same overview screen (without having to buy new ones or find a bunch of phones).

**Comment by calvin-smith** [comment]
I also would like to see this added.  The current options for cameras is really limited.

**Comment by drewgt2** [comment]
Me too - would love to integrate my Wyze cam onto dashboard

**Comment by NoSQLKnowHow** [comment]
+1 to this

**Comment by codemonkey2k5** [comment]
Yes please add wyze camera option!

**Comment by andyylin** [comment]
Wyze support would be amazing.

**Comment by m10d** [comment]
hopefully this could be resolved with https://github.com/prusa3d/Prusa-Link-Web/issues/496 (and if it can't, IE wyze cannot offer standard video stream that the likes of v4l can consume - then perhaps this issue should be a lower priority and possibly piggyback later off work supporting open standards).

**Comment by m10d** [comment]
I looked i up - wyze DOES offer RTSP feeds via a different installable (from wyze) firmware.  It's possible wyze supports other standard/open formats that would be trivial & most appropriate to integrate

**Comment by russdogg** [comment]
In regards to RTSP firmware support, Wyze has pulled their official firmware that had that option. You *can* still find those files online elsewhere (but not on Wyze official site). It looks like they will not be supporting this in the future.

**Comment by m10d** [comment]
> In regards to RTSP firmware support, Wyze has pulled their official firmware that had that option. You _can_ still find those files online elsewhere (but not on Wyze official site). It looks like they will not be supporting this in the future.

Well that's unfortunate. 

While its understandable everyone would want "their"  solution (eg wyze cams, for owners of wyze gear) - this just emphasizes why open solutions merit support, as well as prioritization for integration.  

These are not going to decide its not in their "business interest" to allow customers to use a standard integration - ~walled garden wor the win!~

**Comment by Unflexibel** [comment]
Their **could** be a workaround in regards to WebRTC/RTSP/RTMP/HLS for a Wyze Cam

I have a WyzeCam V4 and using Home Automation with 
https://www.home-assistant.io/integrations/prusalink with this add-on: https://github.com/mrlt8/docker-wyze-bridge

maybe useful, maybe not ¯\_( ツ )_/¯
